db/migrate/20230503192155_sync_log_has_unique_state.rb

```diff
@@ -0,0 +1,14 @@
+class SyncLogHasUniqueState < ActiveRecord::Migration[7.0]
+  def change
+    reversible do |dir|
+      dir.up do
+        SyncLog.where.not(id: SyncLog.latest).delete_all
+      end
+    end
+
+    add_index :sync_logs, [:feeder_type, :feeder_id], unique: true
+
+    add_column :sync_logs, :api_response, :text
+    rename_column :sync_logs, :sync_completed_at, :updated_at
```

Comment:
There was no `updated_at` until now?  Always good to have the created/updated timestamps!